install.sh

```diff
@@ -80,96 +100,106 @@ if [[ $arch == "x86_64" ]]; then
-    if [[ $python_version == "3.6"* ]]; then
-        wget -O - $PIP_UPDATE_URL_36 | python3
-    else
-        wget -O - $PIP_UPDATE_URL | python3
-    fi
+    if [[ $jetpack_release == "35" ]]; then  # JetPack 5.x
+        if ! [[ "$python_version" == "3.8"* ]]; then
+            printf "Unsupported python version. Abort.\n"
+            exit 1
+        fi
 
-    # Install additional dependecies.
-    python3 -m pip install sympy packaging six protobuf~=3.0
-
-    if [[ $jetpack_revision == "7.2" || $jetpack_revision == "7.1" ]]; then
-        if [[ $python_version == "3.6"* ]]; then
-            python3 -m pip install $ORT_PY36_AARCH64_JP461_WHL_URL
-        elif [[ $python_version == "3.7"* ]]; then
-            python3 -m pip install $ORT_PY37_AARCH64_JP461_WHL_URL
-        elif [[ $python_version == "3.8"* ]]; then
-            python3 -m pip install $ORT_PY38_AARCH64_JP461_WHL_URL
+        # Update pip to latest version.
+        python3 -m pip install --upgrade pip
+
+        if [[ $jetpack_revision == "1.0" ]]; then  # JP 5.0.2 (35.1.0)
+            python3 -m pip install cuda-python~=11.6.0
+            python3 -m pip install $ONNXOPTIMIZER_PY38_AARCH64_WHL_URL
+            python3 -m pip install $TRT_PY38_AARCH64_JP_35_1_0_WHL_URL
+            python3 -m pip install --force-reinstall $ORT_PY38_AARCH64_JP_35_1_0_WHL_URL
         fi
-    elif [[ $jetpack_revision == "6.1" ]]; then
-        if [[ $python_version == "3.6"* ]]; then
-            python3 -m pip install $ORT_PY36_AARCH64_JP46_WHL_URL
-        elif [[ $python_version == "3.7"* ]]; then
-            python3 -m pip install $ORT_PY37_AARCH64_JP46_WHL_URL
-        elif [[ $python_version == "3.8"* ]]; then
-            python3 -m pip install $ORT_PY38_AARCH64_JP46_WHL_URL
+
+        # Install additional dependecies.
+        python3 -m pip install six protobuf~=3.0
+
+    elif [[ $jetpack_release == "32" ]]; then  # JetPack 4.x
+        if ! [[ "$python_version" == "3.6"* ]]; then
+            printf "Unsupported python version. Abort.\n"
+            exit 1
+        fi
+
+        # Update pip to latest version.
+        wget -O - $PIP_UPDATE_URL_36 | python3
+
+        # Install ONNX Optimizer.
+        python3 -m pip install onnx~=1.11.0
+        python3 -m pip install $ONNXOPTIMIZER_PY36_AARCH64_WHL_URL
+
+        if [[ $jetpack_revision == "5.1" ]]; then  # JP 4.5.1 (32.5.1)
+            python3 -m pip install --force-reinstall $ORT_PY36_AARCH64_JP_32_5_1_WHL_URL
+        elif [[ $jetpack_revision == "6.1" ]]; then  # JP 4.6.0 (32.6.1)
+            python3 -m pip install --force-reinstall $ORT_PY36_AARCH64_JP_32_6_1_WHL_URL
+        elif [[ $jetpack_revision == "7.1" ]]; then  # JP 4.6.1 (32.7.1)
+            python3 -m pip install --force-reinstall $ORT_PY36_AARCH64_JP_32_7_1_WHL_URL
+        elif [[ $jetpack_revision == "7.2" ]]; then  # JP 4.6.2 (32.7.2)
+            python3 -m pip install --force-reinstall $ORT_PY36_AARCH64_JP_32_7_2_WHL_URL
```

Comment:
I think it is good idea to add else section. The "else" section should protect us from script inconsistency.